azure-iot-device/azure/iot/device/common/mqtt_transport.py

```diff
@@ -49,7 +49,7 @@
 
 # Default keepalive.  Paho sends a PINGREQ using this interval
 # to make sure the connection is still open.
-DEFAULT_KEEPALIVE = 60
+DEFAULT_KEEPALIVE = 180
```

Comment:
In my view, this default value should be maintained within the config object, so all our defaults are in one place.

Reply:
Or, does this come from constant.py?  the max keepalive is in there, why not the default?

---
In reply to: [461186383](https://github.com/Azure/azure-iot-sdk-python/pull/635#discussion_r461186383) [](ancestors = 461186383)

Reply:
Config objects have default values for if the value is not provided with the argument syntax of a function. So I would think the default value would be there

---
In reply to: [461196312](https://github.com/Azure/azure-iot-sdk-python/pull/635#discussion_r461196312) [](ancestors = 461196312,461186383)